vagrant/ansible/upload.yaml

```diff
@@ -15,44 +15,6 @@
   - set_fact: gopath="{{ lookup('env','GOPATH') }}"
   - debug: msg="gopath {{gopath}}"
 
-  - name: Creating /vagrant/installer
```

Comment:
Can you also provide a bit of detail why this is being removed?

Reply:
This file has duplicate code which is not needed here